starters/vue-starter-ts/pages/articles/[id].vue

```diff
@@ -1,64 +1,33 @@
 <script setup lang="ts">
-import type { Article } from '@pantheon-systems/pcc-vue-sdk'
-import { ArticleRenderer } from '@pantheon-systems/pcc-vue-sdk/components'
-import LeadCapture from '~/components/smart-components/LeadCapture.vue'
+import ArticlePreview from '../../components/article/ArticlePreview.vue';
+import ArticleView from '../../components/article/ArticleView.vue';
+import type { PantheonClientConfig } from '@pantheon-systems/pcc-vue-sdk/*';
 
 const route = useRoute()
-const articleId = route.params.id as string
+const config = useRuntimeConfig()
 
+const pccGrant = useCookie('PCC-GRANT')
```

Comment:
I think we should also try to take this from the URL's query. The copy URL feature isn't working on sites whose article pages only take pccGrant from the cookie.